all data/src/battlepass_runner.py

```
@@ -0,0 +1,131 @@
+
+#Not gonna lie, I have no clue what most of the code above this comment does. Im pretty all of it, save logging, can be removed
+#But my code works, so im not gonna bother with that
+
+
+
+
+def navigate_to_battlepass():
+    """
+    Navigate to Battlepass missions from the main menu.
+    """
+    logger.info(f"Navigating to Battlepass")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)